README.md

```diff
@@ -227,7 +226,9 @@ db/0/sqlite
   - As of 2.18, this is a height of 500. It takes four hours or so to reach this from scratch.
   - The rest api should be exposed on your localhost, so you can use the command below to check the height of chain 0.
 
-    `curl -s http://localhost:8080/chainweb/0.0/development/cut | jq '.hashes."'0'".height'`
+    `curl -s http://localhost:8080/chainweb/0.0/fast-development/cut | jq '.hashes."'0'".height`
```

Comment:
I left the 8080:80 port mapping in docker-compose.yml as it was, so nginx can be reached on port 8080 from outside of the docker network.
An alternative to my change would have been to update the port mapping to 8080:8080 instead of updating the nginx config.